src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.epics.js

```diff
@@ -122,3 +124,36 @@ export const openEnrollmentPageEpic = (action$: InputObservable, store: ReduxSto
         },
         ),
     );
+
+export const clickLinkedRecordEpic = (action$: InputObservable, store: ReduxStore, { history }: ApiUtils) =>
+    action$.pipe(
+        ofType(enrollmentPageActionTypes.LINKED_RECORD_CLICK),
+        flatMap(({ payload }) => {
+            let url;
+            const { programId, orgUnitId } = payload;
+            if (payload.eventId) {
+                const recordProgram = getProgramFromProgramIdThrowIfNotFound(programId);
+                if (recordProgram instanceof EventProgram) {
+                    url = `/viewEvent?viewEventId=${payload.eventId}`;
+                } else {
+                    url = `/enrollmentEventEdit?${buildUrlQueryString({
```

Comment:
Hey, 
I believe the URL is missing the  `stageId`, `teiId` and `enrollmentId` parameters. There are some errors when loading the page `enrollmentEventEdit` without all the parameters. 

<img width="1678" alt="Screenshot 2022-05-17 at 08 52 51" src="https://user-images.githubusercontent.com/6817073/168747862-6ac03497-99de-4184-8141-10abf48a827f.png">

Thanks!

Reply:
hey I had a talk with @JoakimSM about this and we agree user should be able to navigate to this page with only `eventId`. Joakim will make a ticket for it separately. Thanks